client/app/components/packages/pas-form/edit.hbs

```diff
@@ -286,6 +288,7 @@
               <saveable-form.radio
                 @value={{radio.code}}
                 @groupValue={{saveable-form.saveableChanges.dcpProjectareaindustrialbusinesszone}}
+                @changed={{fn this.updateAttr saveable-form.saveableChanges "dcpProjectareaindutrialzonename" ""}}
```

Comment:
This is a clever solution to use the `changed` property and set the value to null on change!